ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -95,8 +95,13 @@ EOF
     fi
 
     # CORCI-1096
+    grep 'relayhost' /etc/postfix/main.cf
     echo 'relayhost = [mail.wolf.hpdd.intel.com]' >> /etc/postfix/main.cf
-    postfix reload
+    if ! postfix reload; then
```

Comment:
It may also be useful to enable some verbose output here:
```suggestion
    if ! postfix -v reload; then
```